scripts/run_single_query_and_check.sh

```
@@ -0,0 +1,47 @@
+if [ "$#" -lt 5 ]; then
+    echo "Usage: $0 <executable> <query_file> <declaration> <csv> <expected_output_file> [optional_args...]"
+    exit 1
+fi
+
+executable=$1
+query_file=$2
+declaration=$3
+csv=$4
+expected_output_file=$5
+shift 5
+optional_args="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)